content/en/Platform Deep Dive/Pentests/Reports/customize-report.md

```diff
@@ -7,37 +7,29 @@ description: >
 ---
 
 {{% pageinfo %}}
-This feature may be limited to subscribers with a specific {{% ptaas-tier %}}.
+If allowed by your {{% ptaas-tier %}}, you can customize reports for [Comprehensive Pentests](/getting-started/glossary/#comprehensive-pentest).
 {{% /pageinfo %}}
 
-You can customize the following report types for [Comprehensive Pentests](/getting-started/glossary/#comprehensive-pentest):
-
-- Customer Letter
-- Attestation Report
-- Full Report
-- Full Report + Finding Details
-
-You can't customize an [Attestation Letter](/getting-started/glossary/#attestation-letter) (for [Comprehensive Pentests](/getting-started/glossary/#comprehensive-pentest)) and [Automated Report](/getting-started/glossary/#automated-report) (for [Agile Pentests](/getting-started/glossary/#agile-pentest)).
-
-In our application, we make pentest reports available when they're ready for 
+We make pentest reports available when they're ready for 
 [Remediation](/platform-deep-dive/pentests/pentest-process/pentest-states/), or when they're [Closed](/platform-deep-dive/pentests/pentest-process/pentest-states/).
 To find and customize what you see in a report, take the following steps:
 
-1. In the Cobalt app, select **Pentests**, select the **State** list, and select **Remediation**.
+1. In the Cobalt app, select **Pentests**, and then select **Remediation** or **Closed** in the **State** list.
 1. Select your Comprehensive Pentest, and select the **Report** tab. You'll see report sections,
    along with the **Report** list that allows you to select from the [pentest report
    types](/platform-deep-dive/pentests/reports/#pentest-report-types).<br>
    ![Report list on the pentest page](/deepdive/PentestReportTypes.png "Select a report type for your pentest")
 1. You can now customize the selected report type. Select **Customize**, and scroll to a
    report section.
-   - You can start from any of the other report templates, except the Attestation Letter.
 1. If you want to leave out a report section, select the eye icon next to the section title.
-   As we report all findings, we do not allow you to leave out any finding details.<br>
-   ![Customize a pentest report](/deepdive/CustomizePentestReport.png "Select the eye icon to leave a section out of a pentest report.")
-1. When you've finished customizing your report, scroll to the top of the page and select **Apply**.
+    - You can't hide the [Executive Summary](/platform-deep-dive/pentests/reports/report-contents/#executive-summary).
+    - For an Attestation Letter, section titles don't appear. You can only hide an overall list of findings.<br>
```

Comment:
I would not call the table as list of findings, instead it can be changed to call "findings summary table". This can be different from "Summary of Findings".